tasks/copy_to_glacier/README.md

```diff
@@ -192,29 +192,45 @@ The output of this lambda is a dictionary with a `granules` and `copied_to_glaci
 }
 ```
 
-## pydoc request_files
+## pydoc copy_to_glacier
 
 ```
 NAME
     copy_to_glacier
 
 FUNCTIONS
-    copy_granule_between_buckets(source_bucket_name: str, source_key: str, destination_bucket: str, destination_key: str) -> None
+    copy_granule_between_buckets(source_bucket_name: str, source_key: str, destination_bucket: str, destination_key: str, multipart_chunksize_mb: int) -> None
```

Comment:
should `multipart_chunksize_mb` be of type `int` or `float`?

Reply:
Float. Fixed.

Reply:
int now